MyTestGen.sublime-snippet

```diff
@@ -66,14 +66,38 @@ void strmx(int mxlen, int n) {
 } 
 
 void tree(int n) {
-	int exc = rand(1,n);
-	for (int i = 1; i <= n; ++i)
-	{
-		if(i==exc) continue;
-		int first = rand(1,n);
-		while(first==i) first = rand(1,n);
-		cout<<first<<" "<<i<<endl;
-	}
```

Comment:
this part of code sometimes generates symmetric pairs i.e (3,7) and (7,3) which is now fixed